eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/io/DbsnpVariantsWriterTest.java

```diff
@@ -289,35 +313,88 @@ public void repeatedClusteredVariants() throws Exception {
                                                                   allelesMatch, DEFAULT_VALIDATED);
         DbsnpSubmittedVariantEntity submittedVariantEntity1 = new DbsnpSubmittedVariantEntity(
                 SUBMITTED_VARIANT_ACCESSION_1, hashingFunctionSubmitted.apply(submittedVariant1), submittedVariant1, 1);
         DbsnpVariantsWrapper wrapper1 = buildSimpleWrapper(Collections.singletonList(submittedVariantEntity1));
 
-        SubmittedVariant submittedVariant2 = defaultSubmittedVariant();
         DbsnpSubmittedVariantEntity submittedVariantEntity2 = new DbsnpSubmittedVariantEntity(
-                SUBMITTED_VARIANT_ACCESSION_1, hashingFunctionSubmitted.apply(submittedVariant2), submittedVariant2, 1);
+                SUBMITTED_VARIANT_ACCESSION_1, hashingFunctionSubmitted.apply(submittedVariant1), submittedVariant1, 1);
         DbsnpVariantsWrapper wrapper2 = buildSimpleWrapper(Collections.singletonList(submittedVariantEntity2));
 
         DbsnpSubmittedVariantOperationEntity operationEntity1 = createOperation(submittedVariant1);
         wrapper1.setOperations(Collections.singletonList(operationEntity1));
-        DbsnpSubmittedVariantOperationEntity operationEntity2 = createOperation(submittedVariant2);
-        wrapper2.setOperations(Collections.singletonList(operationEntity2));
+        wrapper2.setOperations(Collections.singletonList(operationEntity1));
 
         dbsnpVariantsWriter.write(Arrays.asList(wrapper1, wrapper2));
-        assertClusteredVariantStored(wrapper1);
-        assertClusterVariantDeclusteredStored(wrapper1);
+        assertClusteredVariantStored(0);
+        assertClusteredVariantDeclusteredStored(1, wrapper1);
+    }
+
+    @Test
+    public void multiallelicClusteredVariantsPartiallyDeclustered() throws Exception {
+        boolean allelesMatch = false;
+        SubmittedVariant submittedVariant1 = new SubmittedVariant("assembly", TAXONOMY_2, "project", "contig", START,
+                                                                  "reference", "alternate",
+                                                                  null,
+                                                                  DEFAULT_SUPPORTED_BY_EVIDENCE, DEFAULT_ASSEMBLY_MATCH,
+                                                                  allelesMatch, DEFAULT_VALIDATED);
+        DbsnpSubmittedVariantEntity submittedVariantEntity1 = new DbsnpSubmittedVariantEntity(
+                SUBMITTED_VARIANT_ACCESSION_1, hashingFunctionSubmitted.apply(submittedVariant1), submittedVariant1, 1);
+
+        SubmittedVariant submittedVariant2 = defaultSubmittedVariant();
+        DbsnpSubmittedVariantEntity submittedVariantEntity2 = new DbsnpSubmittedVariantEntity(
+                SUBMITTED_VARIANT_ACCESSION_1, hashingFunctionSubmitted.apply(submittedVariant2), submittedVariant2, 1);
+
+        DbsnpVariantsWrapper wrapper1 = buildSimpleWrapper(Arrays.asList(submittedVariantEntity1,
```

Comment:
there is only one `wrapper` and one `operationEntity` in this test so the `1` is not needed. Applies for this test and the next one.